tsconfig.json

```diff
@@ -1,10 +1,12 @@
 {
   "extends": "@tsconfig/node18/tsconfig.json",
   "compilerOptions": {
+    "module": "ESNext",
```

Comment:
this one was requested by Typescript to support import assertions for JSON modules.
Since bundling with `tsup` it should not be an issue.

Reply:
this probably should be OK
https://www.typescriptlang.org/docs/handbook/modules/reference.html#es2015-es2020-es2022-esnext

Reply:
set back to ES2022 in 2ab45441